### PR TITLE
Reduce redundant testcases

### PR DESCRIPTION
**Pull request by sophia-guo**
Combine cmdLineTester_DataHelperTests_win and
cmdLineTester_DataHelperTests_unix to cmdLineTester_DataHelperTests

Signed-off-by: Sophia Guo <sophiag@ca.ibm.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
@llxia

**Comment by llxia** [comment]
Jenkins test sanity

**Comment by llxia** [comment]
Please do a rebase. https://github.com/eclipse/openj9/pull/1036 changed same playlist.xml

File: test/cmdLineTests/shareClassTests/DataHelperTests/playlist.xml [hunk 1]
```diff
@@ -24,50 +24,18 @@
...
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
-		<testCaseName>cmdLineTester_DataHelperTests_win</testCaseName>
+		<testCaseName>cmdLineTester_DataHelperTests</testCaseName>
 		<variations>
 			<variation>Mode110</variation>
 			<variation>Mode610</variation>	
 		</variations>
-		<command>$(MKTREE) $(REPORTDIR); \
+		<command>$(MKDIR) -p $(REPORTDIR); \
```
Comment (llxia) [inline test/cmdLineTests/shareClassTests/DataHelperTests/playlist.xml]: Should we still use `$(MKTREE) $(REPORTDIR);`?

Comment (sophia-guo) [reply test/cmdLineTests/shareClassTests/DataHelperTests/playlist.xml]: yes, updated.

**Pull request merged**